docs/src/fs.rst

```diff
@@ -301,6 +301,15 @@ API
     .. versionchanged:: 1.10.0 sub-second precission is supported on Windows
     .. versionchanged:: 2.0.0 replace uv_file with uv_os_fd_t
 
+.. c:function:: int uv_fs_utime_ex(uv_loop_t* loop, uv_fs_t* req, const char* path, double btime, double atime, double mtime, uv_fs_cb cb)
+.. c:function:: int uv_fs_futime_ex(uv_loop_t* loop, uv_fs_t* req, uv_file file, double btime, double atime, double mtime, uv_fs_cb cb)
+
+    Equivalent to :c:func:`uv_fs_utime` and :c:func:`uv_fs_futime` unless you
+    are on OS X or Windows in which case these variants allow you to also set
```

Comment:
Should we mention both?  I mean, _"OS X"_ valid prior to the last couple releases _(not sure which it changed off the top of my head)_.  I could see: _"macOS/OS X"_

Reply:
I think "macOS" has been the way to go since mid 2016 - http://fortune.com/2016/06/13/apple-mac-os-sierra/. I don't have strong opinions on this though.

Reply:
Works for me.  Thanks for the quick response.